lstchain/image/pdf.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import numexpr as ne
+
+
+def log_gaussian(x, mean, sigma):
+    """
+    Evaluate the log of a normal law
+
+    Parameters
+    ----------
+    x: float or array-like
+        Value at which the log gaussian is evaluated
+    mean: float
+        Central value of the normal distribution
+    sigma: float
+        Width of the normal distribution
+
+    Returns
+    -------
+    log_pdf: float or array-like
+        Log of the evaluation of the normal law at x
+
+    """
+    pi = np.pi
+    assert pi
```

Comment:
Okay, story time: What terrible thing happened that you added a check that `numpy.pi` is not 0?

Reply:
I implemented the use of numexpr to improve the code speed by around 10% but the variables used in numexpr.evaluate are not considered as used by pyflakes. So the CI fails due to pi here and log_gauss (in reconstructor.py l637).
So this is kind of a dirty hack that I would be happy to remove but didn't find a solution yet (also tried adding inline comment NOQA but it didn't work)

Reply:
AAAAH, ok. Makes sense, kind of. A better solution would be to pass the `local_dict` option, which is also safer:

```
ne.evaluate("-(x - mean) ** 2 / (2 * sigma ** 2) - log((sqrt(2 * pi) * sigma))", local_dict=dict(x=x, mean=mean, sigma=sigma, pi=np.pi))
```

NOQA only works with `flake8` I think, not pyflakes.

Reply:
Thanks I will implement this.